setup.py

```diff
@@ -3,23 +3,38 @@
 from setuptools import setup, find_packages
 
 
-with open('README.rst') as f:
+with open('README.md') as f:
     readme = f.read()
 
 with open('LICENSE') as f:
     license = f.read()
 
+install_requires = ['setuptools==36.2.7',
+                    'hvac==0.2.17+git.b817da4',
+                    'requests==2.18.4',
```

Comment:
Don't pin these versions if possible (I guess except for `hvac`)

Reply:
Unpinned all except `hvac`.